website/src/wordpress-page.tsx

```diff
@@ -56,7 +67,21 @@ const parseOptions: HTMLReactParserOptions = {
           {domToReact(node.children, parseOptions)}
         </Link>
       )
-    }
+    } /*  else if (
+      "name" in node &&
+      "attribs" in node &&
+      node.attribs &&
+      node.attribs.startsWith?(wordpressUrl)
+    ) {
+      console.log(node.name)
+    } */
     return undefined
   },
 }
+function pullWords(
+  arg0: string | undefined,
+  arg1: string | undefined,
+  arg2?: string | undefined
+): false | void | object | JSX.Element | null | undefined {
+  throw new Error("Function not implemented.")
+}
```

Comment:
What's up with the typing on this? I'd say just define a very simple React component which renders the parts of the reference to demonstrate that they're properly captured. Say, transforming `[WJ20:1-5]` into `Words #1 through #5 in WJ20` or something, until we hook up the actual content. But we don't want to throw an error as our stop-gap.